sonar-csharp-plugin/pom.xml

```
@@ -213,8 +213,8 @@
             <configuration>
               <rules>
                 <requireFilesSize>
-                  <maxsize>5400000</maxsize>
-                  <minsize>5200000</minsize>
+                  <maxsize>5410000</maxsize>
+                  <minsize>5300000</minsize>
```

We're bumping the higher digit, see #5447
```suggestion
                <maxsize>5500000</maxsize>
                  <minsize>5300000</minsize>
```